src/js/facility-locator/containers/FacilityLocatorApp.jsx

```diff
@@ -1,10 +1,46 @@
+import { Link } from 'react-router';
 import React from 'react';
 
 class FacilityLocatorApp extends React.Component {
+  renderBreadcrumbs() {
```

Comment:
These breadcrumbs are specific to the Facility Locator. 

`Home > Facility Locator > Facility Details`

The Facility Details link is owned by the Locator and I could not find an easy way for MetalSmith to know that we've navigated into a detail page so I made my own "breadcrumbs" here